src/utils/legend.coffee

```diff
@@ -107,8 +108,8 @@
             item.append('text')
               .attr(
                 'class': (d, i) -> "legendText series_#{i}"
-                'font-family': 'Courier'
-                'font-size': 10
+                'font-family': options.fontFamily
+                'font-size': options.fontSize
```

Comment:
We are adding loads of stuff here to `attr` that should better go to `style`; could you change this please